test/dotnet.Tests/project.json

```diff
@@ -43,6 +43,7 @@
     ]
   },
   "buildOptions": {
+    "keyFile": "../../tools/Key.snk",
```

Comment:
The other projects I checked didn't:

https://github.com/dotnet/cli/blob/rel/1.0.0/test/dotnet-compile.UnitTests/project.json#L4
https://github.com/dotnet/cli/blob/rel/1.0.0/test/Microsoft.DotNet.Configurer.UnitTests/project.json#L4